api-admin/src/main/java/bio/terra/pearl/api/admin/service/admin/AdminTaskExtService.java

```diff
@@ -0,0 +1,71 @@
+package bio.terra.pearl.api.admin.service.admin;
+
+import bio.terra.pearl.api.admin.service.AuthUtilService;
+import bio.terra.pearl.core.model.EnvironmentName;
+import bio.terra.pearl.core.model.admin.AdminUser;
+import bio.terra.pearl.core.model.participant.Enrollee;
+import bio.terra.pearl.core.model.study.PortalStudy;
+import bio.terra.pearl.core.model.study.StudyEnvironment;
+import bio.terra.pearl.core.model.workflow.AdminTask;
+import bio.terra.pearl.core.model.workflow.DataAuditInfo;
+import bio.terra.pearl.core.service.study.StudyEnvironmentService;
+import bio.terra.pearl.core.service.workflow.AdminTaskService;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AdminTaskExtService {
```

Comment:
currently, yes, tasks just hang around as "completed".  The idea is that it's kind of like Jira -- you don't delete tasks, you just mark them as done and they (hopefully) fade from view.  Obviously, the UI on this isn't polished yet, but that's the idea.  I think you make a good case for the accidental "oops, undo that" task creation, although in audited environments like ours, it's sometimes cleaner to just mark "oopses" as "complete" or "rejected" rather than having to manage a separate delete workflow.